### ISSUE TITLE
CustomFields of schema classes are not replicated in distributed environment

### ISSUE THREAD

**Issue by ferrisbuhler** [orientechnologies/orientdb#9524]
### OS: Linux (Docker)

Some OrientDB servers (in our case: 3) are connected in distributed mode.
One client being connected to server A creates a schema class with a property and writes data into the "customFields" of each the class and the property.

## Expected behavior
Another client being connected to server B sees the class with its property and can retrieve the data in the "customFields" of both.

## Actual behavior  
Clients connected  to server B are able to retrieve the "customFields" of the property, but **NOT** the "customFields" of the class.

## Steps to reproduce
```java
void createSchema() {
  try(ODatabaseSession session = orientDbService.getSession()) {
    oClass = session.createVertexClass("myClass");
    oClass.createProperty("myProperty");
  }
}

void writeCustom() {
  OClass oClass = session.getMetadata().getSchema().getClass("myClass");
  oClass.setCustom("myKey", "myData");
  OProperty oProperty = oClass.getProperty("myProperty");
  oProperty.setCustom("myKey", "myData");
}

@Test
void testReadClassCustom() {
  OClass oClass = session.getMetadata().getSchema().getClass("myClass");
  String data = oClass.getCustom("myKey");
  Assertions.assertNotNull(data);
  Assertions.assertEquals("myKey", data);
}
@Test
void testReadPropertyCustom() {
  OClass oClass = session.getMetadata().getSchema().getClass("myClass");
  OProperty oProperty = oClass.getProperty("myProperty");
  String data = oProperty.getCustom("myKey");
  Assertions.assertNotNull(data);
  Assertions.assertEquals("myKey", data);
}
```
On client A:
- run createSchema();
- run writeCustom();
- run testReadClassCustom();  // <- succeeds
- run testReadPropertyCustom();  // <- succeeds

On client B:
- run testReadClassCustom();  // <- **fails**
- run testReadPropertyCustom();  // <- succeeds

**Comment by lvca** [comment]
@tglman is this fixed in 3.2.x?

**Comment by tglman** [comment]
Hi,

This is implemented properly in 3.2.x and 3.1.x, closing, please re-open if you still experience some problems.

Bey
